fft/src/KokkosFFT_Plans.hpp

```diff
@@ -271,23 +269,86 @@ class Plan {
   Plan& operator=(Plan&&) = delete;
   Plan(Plan&&)            = delete;
 
+  /// \brief Execute FFT on input and output Views with normalization
+  ///
+  /// \param in [in] Input data
+  /// \param out [out] Ouput data
+  void execute(const InViewType& in, const OutViewType& out,
```

Comment:
documentation of `norm` seems to be missing.

Reply:
Fixed accordingly.